truenas/webui
# NAS-123838 / 24.04.0 / Add support for legacy model ES24N (additions)

**Testing**

1. Connect to machine specified in the ticket's commentaries
2. Go to **System Settings > Enclosure** page, and select **ES24N** from the right sidebar 
3. Verify visibility of **Identify Drive** button for model of enclosure (`ES24N`). For this enclosure, on this particular model, button should be hidden, since it is not supported in Dragonfish.

4. Verify the enclosure image displayed on **Dashboard** page at **System Information** widget. **Expected image:**  
<img width="293" alt="image" src="https://github.com/truenas/webui/assets/20611516/e0612c68-5751-41d4-89d1-8ce5696a2a3e">
